src/main/java/org/beanpod/switchboard/controller/LogController.java

```diff
@@ -51,4 +56,14 @@
         .map(ResponseEntity::ok)
         .orElseThrow(() -> new ExceptionType.UnknownException(CONTROLLER_NAME));
   }
+
+  @Override
+  public ResponseEntity<StreamLogModel> createStreamLog(
+      CreateStreamLogRequest createStreamLogRequest) {
+    return Optional.of(createStreamLogRequest)
+        .map(streamLogService::createLog)
+        .map(logStreamMapper::toStreamLogModel)
```

Comment:
I just realized we should rename `LogStreamMapper` to `StreamLogMapper`

Reply:
Good catch! I rename the class